hack/testing/rsyslog/roles/rsyslog/defaults/main.yaml

```diff
@@ -2,16 +2,35 @@
 # General configuration
 # ---------------------
 
-# paths
-rsyslog__viaq_path: /etc/rsyslog.d/viaq
-rsyslog__viaq_log_path: /var/log/containers
-
 # .. envvar:: rsyslog__enabled
 #
 # Enable or disable ``rsyslog`` management.
 rsyslog__enabled: True
 
-rsyslog__work_directory: /var/lib/rsyslog
+# .. envvar:: rsyslog__home
+#
+# Top directory to place logs
+rsyslog__home: '{{ "/var/log" }}'
```

Comment:
No brackets required .
It can be

rsyslog_home: '/var/log'

Also for all the other params like 
rsyslog_dir, rsyslog_work_directory and so on...

Reply:
Yep, you are right.  It's mistakenly brought in by the porting effort of debops...  Fixing them.